go.mod

```diff
@@ -5,6 +5,29 @@ go 1.19
 require github.com/spf13/cobra v1.7.0
```

Comment:
Seems non-transitive dependencies are missing here.

Run `go mod tidy` to get a new require block of:

```
require (
        github.com/deepmap/oapi-codegen v1.12.4
        github.com/getkin/kin-openapi v0.116.0
        github.com/labstack/echo/v4 v4.10.2
        github.com/spf13/cobra v1.7.0
)
```